### PR TITLE
Teams bug fix

### PR DESCRIPTION
**Pull request by mhofert-r7**
## Requirements
Update Workflows to new version of teams

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### CODE REVIEW

File: workflows/Open_Source_Indicator_Enrichment_with_Microsoft_Teams/help.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Plugins utilized by workflow:
 |----|----|--------|
 |Dig|1.0.3|1|
 |Team Cymru MHR|1.0.3|1|
-|Microsoft Teams|2.0.2|6|
+|Microsoft Teams|2.0.4|7|
```
Comment (mrinehart-r7) [inline workflows/Open_Source_Indicator_Enrichment_with_Microsoft_Teams/help.md:59]:
```suggestion
|Microsoft Teams|2.0.4|7|
```
Did you mean to increment the usage count?

File: workflows/Post_Gmail_Phishing_Alerts_to_Microsoft_Teams/workflow.spec.yaml [hunk 2]
```diff
@@ -25,5 +25,5 @@ resources:
     title: Example Message to Microsoft Teams
   - name: artifact1.png
     title: Top Level Headers Artifact
-  - name: artifact1.png
+  - name: artifact2.png
```
Comment (mrinehart-r7) [inline workflows/Post_Gmail_Phishing_Alerts_to_Microsoft_Teams/workflow.spec.yaml:28]: Does this workflow validate properly now?

File: workflows/Open_Source_Indicator_Enrichment_with_Microsoft_Teams/help.md (hunk 1 above)
Comment (mhofert-r7) [reply workflows/Open_Source_Indicator_Enrichment_with_Microsoft_Teams/help.md:59]: Yep it was failing validation.

File: workflows/Post_Gmail_Phishing_Alerts_to_Microsoft_Teams/workflow.spec.yaml (hunk 2 above)
Comment (mhofert-r7) [reply workflows/Post_Gmail_Phishing_Alerts_to_Microsoft_Teams/workflow.spec.yaml:28]: yes

**Pull request merged**